src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  ComponentFixture,
+  async,
+  TestBed,
+} from '@angular/core/testing';
+import {Component, DebugElement} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {
+  CdkOption,
+  CdkListboxModule
+} from './index';
+import {dispatchMouseEvent} from '@angular/cdk/testing/private';
+
+describe('CdkOption', () => {
+
+  describe('selection state change', () => {
+    let fixture: ComponentFixture<CdkListboxWithCdkOptions>;
+    let options: DebugElement[];
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkListboxModule],
+        declarations: [CdkListboxWithCdkOptions],
+      }).compileComponents();
+    }));
+
+    beforeEach(async(() => {
+      fixture = TestBed.createComponent(CdkListboxWithCdkOptions);
+      fixture.detectChanges();
+      options = fixture.debugElement.queryAll(By.directive(CdkOption));
```

Comment:
Rather than repeating `.nativeElement` and `.injector.get<CdkOption>(CdkOption)` below, you can just do it once here:
```ts
optionInstances = options.map(o => o.injector.get(CdkOption));
optionElements = options.map(o => o.nativeElement);
```

Reply:
Oh I see now. Definitely much better :)